vrecord

```diff
@@ -348,6 +356,91 @@ _master_gui(){
+                                                        label="TBC Model" text="Enter TBC Model"
+        _construct_pashua_id CC_TBC_Serial              x=340 y=155 type=textbox height=30 \
+                                                        label="TBC Serial Number" text="Enter TBC Serial Number"
+        _construct_pashua_id CC_ADC_MAKE                x=670 y=300 type=textbox height=30 \
+                                                        label="Analog-Digital Converter Make" text="Enter Analog-Digital Converter Make"
+        _construct_pashua_id CC_ADC_Model               x=670 y=235 type=textbox height=30 \
+                                                        text="Enter Analog-Digital Converter Model" label="Analog-Digital Converter Model" 
+        _construct_pashua_id CC_ADC_Serial              x=670 y=155 type=textbox height=30 \
+                                                        label="Analog-Digital Converter Serial Number" text="Enter Analog-Digital Converter Serial Number"
+        _construct_pashua_id CC_NOTES                   x=50 y=20 type=textbox height=100 width=400 \
+                                                        label="Notes:"
+    }
+        _construct_capture_config_gui > "${PASHUA_CONFIGFILE}"
+        _pashua_run
+
+        if [[ "${cb}" = 0 ]] ; then
+
+            if  [[ ! -d "${CC_PROFILE_DIR}" ]] ; then
+                CC_PROFILE_DIR="${CC_PROFILE_DIR_Default}"
+                if [[ ! -d "${CC_PROFILE_DIR_Default}" ]] ; then
+                    mkdir "${CC_PROFILE_DIR_Default}" 
+                    CC_PROFILE_DIR="${CC_PROFILE_DIR_Default}"
+                fi
+            fi
+            
+        CAPTURE_CONFIGURATION_PROFILE=""$(echo ${CC_PROFILE_DIR})"/"$(echo ${CC_PROFILE_NAME})""
+            
+            echo
+            echo "path is "$(echo ${CC_PROFILE_DIR} | sed 's/ /\\\ /g')""
+            echo "name is ${CC_PROFILE_NAME}"
+            echo "profile is "$(echo "${CAPTURE_CONFIGURATION_PROFILE}" | sed 's/ /\\\ /g')" .txt"
+            echo
+
+            {
+                echo "# Set these variables to a valid option or leave as empty quotes (like \"\") to request each run."
+                for COMMENT_LINE in CC_PROFILE_DIR CC_PROFILE_NAME CC_DECK_MAKE CC_DECK_MODEL CC_DECK_Serial \
+                CC_TBC_MAKE CC_TBC_MODEL CC_TBC_Serial CC_ADC_MAKE CC_ADC_Model CC_ADC_Serial CC_NOTES ; do
+                    echo "${COMMENT_LINE}=\"${!COMMENT_LINE}\""
+                done
+                echo "Profile Name: ${CC_PROFILE_NAME}"
+                echo "Profile Path "$(echo "${CAPTURE_CONFIGURATION_PROFILE}" | sed 's/ /\\\ /g')".txt"
+            } > "${CAPTURE_CONFIGURATION_PROFILE}".txt
+            
+            rm "${PASHUA_CONFIGFILE}"
+            _edit_mode
+            
+        else rm "${PASHUA_CONFIGFILE}"
+            _edit_mode
+             
+        fi
+        if [[ -e "${PASHUA_CONFIGFILE}" ]] ; then
+            rm "${PASHUA_CONFIGFILE}"
+        fi
```

Comment:
Nit: Please fix the indentation of lines 400 to 440 as well. (I personally would add an empty line between 399 and 400 and would avoid two empty lines at 363 and 364.)